doc/source/building/build_live_iso.rst

```diff
@@ -60,9 +61,9 @@ system from stick do the following:
 
       $ dd if=LimeJeOS-Leap-42.1.x86_64-1.42.1.iso of=/dev/<stickdevice>
 
-3. Run system from stick
+6. Start system from USB stick
 
-   Like in the example above use Qemu to run the system, but attach
+   Like in the example above use QEMU to start the system, but attach
```

Comment:
I've provided another commit (see 03cafdc) were you can see high- and low-level in place.

I've introduced a "Working with Images" section where we can put all of these "low-level topics". Currently, this section is placed under "Building Images"; I'm not sure if we should do that or move it to an upper level.

Let me know what you think about. 😉